package.json

```diff
@@ -31,19 +31,24 @@
     "yaml": "^1.10.0"
   },
   "devDependencies": {
+    "@types/glob": "^7.1.3",
     "@types/jest": "^26.0.19",
-    "@types/node": "^14.14.9",
-    "@types/minimatch": "^3.0.0",
     "@types/js-yaml": "^3.12.1",
+    "@types/minimatch": "^3.0.0",
+    "@types/node": "^14.14.14",
     "@typescript-eslint/parser": "^4.10.0",
     "@vercel/ncc": "^0.25.1",
+    "directory-tree": "^2.2.5",
     "eslint": "^7.13.0",
+    "eslint-config-prettier": "^7.0.0",
     "eslint-plugin-github": "^4.1.1",
     "eslint-plugin-jest": "^24.1.3",
+    "glob": "^7.1.6",
     "jest": "^24.9.0",
     "jest-circus": "^26.4.2",
     "js-yaml": "^3.13.0",
     "prettier": "2.2.1",
+    "prettier-plugin-organize-imports": "^1.1.1",
```

Comment:
This plugin is handy as it reorders all your imports properly and removes unused imports on the file save. You should have `prettier` extension for this to be installed in your IDE, if you want this to happen.